### DISCUSSION TITLE
how to backup without using docker-compose ?

### DISCUSSION THREAD

**Discussion by fiveangle** [teslamate-org/teslamate#1346] [Q&A]
In many container management frameworks, there is no access to docker-compose.yml (like when using Rancher, Portainer, etc).

Is there any simple method for performing running backups in these cases ?

**Comment by MikeBishop** [comment]
Are you referring to `docker-compose exec -T database pg_dump -U teslamate teslamate > /backuplocation/teslamate.bck` from https://docs.teslamate.org/docs/maintenance/backup_restore/?

That command is just running `pg_dump` in the database container.  You could presumably `docker exec` or equivalent the same command into the same container and get the same result.  The only point of using `docker-compose` here is to map the project-local name (database) to the globally-unique container name.
